pages/Contributing and Debugging/Issue-Guidelines.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Issue Guidelines
+---
+
+Due to the influx of low quality or incomprehensible issues,
+we prefer to begin possible bug reports or feature requests as *discussions*,
+and elevate them to issues if they can be confirmed by a member to be relevant,
```

Comment:
What exactly makes someone a "member" rather than just a user? Does that mean "people in the hyprwm Github org" or "all Hyprland contributers" or something else?

Reply:
in github terms, that's a person in the org.
![image](https://github.com/user-attachments/assets/39f21795-6efa-4d6c-880c-12939c56960b)